### PR TITLE
Update mygene-info.yaml

### PR DESCRIPTION
**Pull request by lwinfree**
finished; removed TODOs

### CODE REVIEW

**Comment by lwinfree** [comment]
please also merge this @kltm. Thanks!

File: data-sources/mygene-info.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id: mygene-info
 source: MyGene.info
 source-link: 'http://mygene.info'
-status: incomplete
+status: complete
 description: 'MyGene.info provides a simple-to-use REST web services to query/retrieve gene annotation data.'
```
Comment (kltm) [inline data-sources/mygene-info.yaml]: @lwinfree Is the description a direct copy? The text I found drops the indefinite article (or maybe they had a typo earlier): `MyGene.info provides simple-to-use REST web services to query/retrieve gene annotation data. `

**Comment by kltm** [comment]
@lwinfree 
Would you mind if I added the following to the commentary (a quote from the license)? I think it is one of the more nasty bits that prevents reuse (i.e. highlighting the scope ding that you give them).

---

MyGene.info may contain information from third party websites, which may or may not be marked with the name of the source. Such information does not necessarily represent the views or opinions of TSRI, and TSRI shall have no responsibility whatsoever for such information.

File: data-sources/mygene-info.yaml (hunk 1 above)
Comment (lwinfree) [reply data-sources/mygene-info.yaml]: can't remember...might be a typo on my end

**Comment by lwinfree** [comment]
@kltm yes please add that commentary! Thanks

**Comment by kltm** [comment]
@lwinfree 
Sorry to keep circling back to this one (I've been talking licenses a lot recently).
I was wondering about the B.1 violation. It think the fact there is no reuse is covered under D and E; my understanding of B.1 is that we'd have to email somebody or send them our analytics; from our explanation: "If the license allows for non-negotiated reuse..."; in this case, it does not so the item cannot be applied and they pass. For comparison, a completely upfront ARR site can get three stars if they are clear. For comparison: omia.

**Comment by lwinfree** [comment]
How is this looking now @kltm? Any other thoughts for Mygene?

**Pull request merged**